Ical.Net.nuspec

```diff
@@ -2,16 +2,14 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
     <metadata>
         <id>Ical.Net</id>
-        <version>2.2.5</version>
+        <version>2.1.5</version>
```

Comment:
This doesn't seem right. I bumped the minor version number from 1 to 2, because I added a couple of public constants a few PRs ago. ical.net is using [semantic versioning](http://semver.org/):

> Given a version number MAJOR.MINOR.PATCH, increment the:
> 
> 1. MAJOR version when you make incompatible API changes,
> 2. MINOR version when you add functionality in a backwards-compatible manner, and
> 3. PATCH version when you make backwards-compatible bug fixes.